tests/large_scale_evaluation.py

```diff
@@ -234,7 +234,7 @@ def main():
 
     results = run_evaluation(model_names, dataset_names, args.max_rows, args.max_columns)
 
-    print(tabulate.tabulate(
+    print(tabulate.tabulate(  # noqa: T001
```

Comment:
I left the print as it seems innocuous, but we could remove it if we really don't want to allow prints.

Reply:
This seems fine to me. @csala What do you think?

Reply:
It's fine, but rather than using the `noqa` here I would use `per-file-ignores`, since the ignore would be valid for the entire file (as it is a user script)